charts/agent-control/Chart.yaml

```diff
@@ -3,15 +3,15 @@ name: agent-control
 description: Bootstraps New Relic' Agent Control
 
 type: application
-version: 0.0.30-beta
+version: 0.0.32-beta
```

Comment:
I guess it is because you have an other PR, but you are skipping a version

Reply:
Yes, this was due to the pending `opamp` -> `fleet_control` change.